pipeline/config/exoscale/sc-config.yaml

```diff
@@ -575,3 +575,15 @@ metricsServer:
   enabled: true
 calicoAccountant:
   enabled: true
+s3Exporter:
+  enabled: true
+  interval: 120s
```

Comment:
Do you think that there are any downsides to having an even higher interval, let's say every 10'th minute?

Reply:
I was also thinking about this. I tried it with 5 minutes and it worked fine, except that Grafana was showing gaps in the graphs. It is probably possible to tweak the dashboard to "know" how sparse the points are, but then this would also need templating if we want to change the interval. Maybe it would be a good idea to lock the interval and configure the dashboard to work with large value?

I should also mention that this is very relevant for sc-logs. It takes a long time to get metrics for this bucket since it contains so many objects. This means that the s3-exporter can start timing out if it takes too long, (and we obviously cannot have the timeout longer than the interval).

Reply:
If you can make the dashboard work with it, then a longer static interval sounds good.